src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -1032,19 +1019,21 @@ public String getImportFullPath() {
         /**
          * Get import start line number.
          *
-         * @return import start line.
+         * @return import ast start line.
          */
         public int getStartLineNumber() {
-            return startLineNumber;
+            return importAST.getLineNo();
         }
 
         /**
          * Get import end line number.
+         * Note: It can be different from <b>startLineNumber</b> when import statement span
```

Comment:
Note should somehow be separated from previous line. Javadoc generation by default will merge these 2 lines into one. I don't know what is the exact standard for notes.

Reply:
Done

Reply:
If I seperate it by a single line I am getting error that empty line should be followed by `<p>` tag.

Link to the error https://ci.appveyor.com/project/Checkstyle/checkstyle/builds/32866348/job/yujeehocq1sxi5hd#L224

Reply:
Previously also, there was no separation between line and note.
Please have a look
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java#L989

Reply:
@rnveach 
I have separated the note using `<p>` tag

Reply:
Done